beta/src/pages/learn/start-a-new-react-project.md

```diff
@@ -1,41 +1,41 @@
 ---
-title: Start a New React Project
+title: एक नया React प्रोजेक्ट शुरू करें
 ---
 
 <Intro>
 
-If you're learning React or considering adding it to an existing project, you can get started quickly by [adding React to any HTML page with script tags](/learn/add-react-to-a-website). If your project will need many components and many files, it might be time to consider the options below!
+यदि आप React सीख रहे हैं या इसे किसी मौजूदा प्रोजेक्ट में इस्तेमाल करने पर विचार कर रहे हैं, तो आप [स्क्रिप्ट टैग को किसी भी HTML पेज पर React इस्तेमाल करके](/learn/add-react-to-a-website) शुरू कर सकते हैं। यदि आपके प्रोजेक्ट को कई कौम्पोनॅन्ट और कई फाइलों की आवश्यकता होगी, तो यह नीचे दिए गए औपशंस पर विचार करने का समय हो सकता है!
 
 </Intro>
 
-## Choose your own adventure {/*choose-your-own-adventure*/}
+## अपना खुद का रोमांच चुनें {/*अपना खुद का रोमांच चुनें*/}
 
-React is a library that lets you organize UI code by breaking it apart into pieces called components. React doesn't take care of routing or data management. For these features, you'll want to use third-party libraries or write your own solutions. This means there are several ways to start a new React project:
+React एक library है जो आपको यूआई कोड को कौम्पोनॅन्ट नामक टुकड़ों में तोड़कर व्यवस्थित करने देता है। React रूटिंग या डेटा प्रबंधन का ध्यान नहीं रखता है। इन सुविधाओं के लिए, आप थर्ड-पार्टी लाइब्रेरी का उपयोग करना चाहेंगे या अपने स्वयं के समाधान लिखना चाहेंगे। इसका मतलब है कि एक नया React प्रोजेक्ट शुरू करने के कई तरीके हैं:
 
-* Start with a **minimal set up with just a toolchain,** adding features to your project as necessary.
-* Start with an **opinionated framework** with common functionality already built in.
+* केवल एक टूलचेन के साथ **न्यूनतम सेट अप के साथ प्रारंभ करें,** आवश्यकतानुसार अपने प्रोजेक्ट में सुविधाएँ जोड़ें।
+* पहले से निर्मित सामान्य कार्यक्षमता के साथ **विचारित ढांचे** के साथ प्रारंभ करें।
 
-Whether you're just getting started, looking to build something big, or want to set up your own toolchain, this guide will set you on the right path.
+चाहे आप अभी शुरुआत कर रहे हैं, कुछ बड़ा बनाना चाहते हैं, या अपनी खुद की टूलचेन स्थापित करना चाहते हैं, यह मार्गदर्शिका आपको सही रास्ते पर ले जाएगी।
 
-## Getting started with a React toolchain {/*getting-started-with-a-react-toolchain*/}
+## React टूलचेन के साथ शुरुआत करना {/*आरंभ करना-के साथ-साथ-प्रतिक्रिया-टूलचेन*/}
```

Comment:
Same as above do not transfer `{/*getting-started-with-a-react-toolchain*/}`

`{/*आरंभ करना-के साथ-साथ-प्रतिक्रिया-टूलचेन*/}` => `{/*getting-started-with-a-react-toolchain*/}`

Reply:
```
## React टूलचेन के साथ शुरुआत करना {/*getting-started-with-a-react-toolchain*/}
```